sql/core/src/main/scala/org/apache/spark/sql/execution/Columnar.scala

```diff
@@ -426,7 +428,8 @@ trait RowToColumnarTransition extends UnaryExecNode
  * populate with [[RowToColumnConverter]], but the performance requirements are different and it
  * would only be to reduce code.
  */
-case class RowToColumnarExec(child: SparkPlan) extends RowToColumnarTransition {
+case class RowToColumnarExec(child: SparkPlan, useArrow: Boolean = false)
```

Comment:
@HyukjinKwon , after I added 4 extra conversion functions in `ArrowConverters`, I felt those can also be used by mapInArrow function, so I only added a parameter here to force specify Arrow as Columnar Backend